app/views/moderations/mod.html.erb

```diff
@@ -24,18 +25,26 @@
   <% elsif current_user.has_role?(:super_admin) && @moderatable.class.name == "Comment" %>
     <h3> <a href="/admin/comments/<%= @moderatable.id %>" data-no-instant>Admin:Comment</a> |
       <a href="/admin/users/<%= @moderatable.user_id %>" data-no-instant>Admin:User</a></h3>
-  <% else %>
-    <p>
-      <b style="font-size:1.3em">All negative reactions are private.</b>
-    </p>
-    <hr>
-    <p>
-      Use <b>Thumbsdown</b> to move this content "down" for any reason (quality, usefulness, etc.).
-    </p>
+  <% end %>
+  <p>
+    <b style="font-size:1.3em">All negative reactions are private.</b>
+  </p>
+  <p>
+    Use <b>thumbsdown</b> to move this content "down" for any reason (quality, usefulness, etc.).
+  </p>
+  <p>
+    Use <b>vomit</b> for code of conduct violations (harassment, being a jerk, spam, etc.).
+  </p>
+  <div class="tag-mod-form">
+    <h2>Flag <a href="<%= @moderatable.user.path %>"><%= @moderatable.user.username %></a> as Abusive</h2>
+    <button class="reaction-button <%= Reaction.cached_any_reactions_for?(@moderatable.user, current_user, "vomit") ? "reacted" : "" %>" data-reactable-id="<%= @moderatable.user.id %>" data-category="vomit" data-reactable-type="<%= @moderatable.user.class.name %>">
```

Comment:
this line is super long :D I suspect you use word wrapping on your editor, right? :D